src/requirements/Sound/components/SoundSupportArtist.tsx

```diff
@@ -0,0 +1,81 @@
+import { FormControl, FormLabel } from "@chakra-ui/react"
+import FormErrorMessage from "components/common/FormErrorMessage"
+import StyledSelect from "components/common/StyledSelect"
+import { useState } from "react"
+import { Controller, useFormContext, useWatch } from "react-hook-form"
+import { RequirementFormProps } from "requirements"
+import useSWRImmutable from "swr/immutable"
+import { SelectOption } from "types"
+
+import parseFromObject from "utils/parseFromObject"
+
+const SupportArtist = ({ baseFieldPath }: RequirementFormProps) => {
+  const {
+    control,
+    formState: { errors },
+  } = useFormContext()
+
+  const [search, setSearch] = useState(
+    useWatch({ name: `${baseFieldPath}.data.id` })
```

Comment:
it's weird to put an useWatch hook as useState's default value. Should get the field prop and put field.data.id here, that's why we have the field prop (in RequirementFormProps)